html/manifest/file_handlers.json

```
@@ -0,0 +1,42 @@
+{
+  "html": {
+    "manifest": {
+      "file_handlers": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/file_handlers",
+          "spec_url": "https://wicg.github.io/manifest-incubations/index.html#file_handlers-member",
+          "support": {
+            "chrome": {
+              "version_added": "102"
+            },
+            "chrome_android": "mirror",
```

```suggestion
            "chrome_android": false,
```

It’s desktop-only so far.